internal-enrichment/virustotal/src/virustotal/virustotal.py

```diff
@@ -267,15 +267,15 @@ def _process_file(self, stix_objects, stix_entity, opencti_entity):
                     result = data["attributes"]["last_analysis_results"][key]
                     content += (
                         "| "
-                        + result["engine_name"]
+                        + str(result.get("engine_name", "N/A") or "N/A")
```

Comment:
If you want to cover the case `result["engine_name"] is None`  I think this suggestion is sufficient

```suggestion
                        + str(result.get("engine_name", "N/A") )
```

because with 
```
 str(result.get("engine_name", "N/A") or "N/A")
``` 
you are overwriting all the "Falsy" results (empty list, empty dict, False, None but also float or int = 0 and empty string)